src/WalterModem.h

```diff
@@ -2874,8 +2885,23 @@ class WalterModem
          * @return The Luhn checksum as an ASCII character.
          */
         static char _getLuhnChecksum(const char *imei);
+        
+        /**
+         * @brief converts a given duration to encoded uint8_t according to the base_times.
+         * 
+         * this function will encode a the given duration according to the base_times / mulipliers for use in PSM.
+         * @warning this is an approximation based on the base_times array.
+         * 
+         * @param base_times pointer to an array containing the base times
+         * @param base_times_len lenght of the base_times array
+         * @param duration_seconds the duration in seconds
+         * @param actual_duration_seconds pointer to where the actual duration in seconds should be filled in 
+         * 
+         * @returns uint8_t converted duration_seconds
```

Comment:
`@return The duration encoded into the 3GPP standard format`. Please use `@return` instead of `@returns` according to Doxygen style.